Makefile

```diff
@@ -84,12 +87,7 @@ test-integration: ## Run integration tests
 .PHONY: test-e2e
 test-e2e: ## Run e2e tests
 	PULL_POLICY=IfNotPresent $(MAKE) docker-build
-	cd $(TEST_E2E_DIR); go test -v -tags=e2e -timeout=3h . -args -ginkgo.v -ginkgo.focus "functional tests" --managerImage $(CONTROLLER_IMG)-$(ARCH):$(TAG)
-
-.PHONY: test-conformance
-test-conformance: ## Run conformance test on workload cluster
-	PULL_POLICY=IfNotPresent $(MAKE) docker-build
-	cd $(TEST_E2E_DIR); go test -v -tags=e2e -timeout=4h . -args -ginkgo.v -ginkgo.focus "conformance tests" --managerImage $(CONTROLLER_IMG)-$(ARCH):$(TAG)
+	go test -v -timeout=3h ./$(TEST_E2E_DIR) -args -ginkgo.v --managerImage $(CONTROLLER_IMG)-$(ARCH):$(TAG)
```

Comment:
capa run this test using golang build tags `+build e2e`, but i found issues in my editor (I thought it was a bug from gopls) which gives a line error even though it's not. so i just specify test path as capz do

Reply:
I'm thinking should we reduce the timeout if we are not running the conformance tests. I think 1 or 2 hours should be enough.

Reply:
updated